webapp.py

```
@@ -63,114 +94,149 @@ def _get_db(name, connect_func):
                 g.db_connections = []
             g.db_connections.append(db)
         return db
 
     @app.teardown_appcontext
     def close_dbs(_error):
         """Closes all database connections at the end of the request."""
-        for conn in getattr(g, 'db_connections', []):
+        for conn in getattr(g, "db_connections", []):
             try:
                 conn.close()
             except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)